doc/tutorials/rtvm_workload_design_guideline.rst

```diff
@@ -0,0 +1,44 @@
+.. _rtvm_workload_guideline:
+
+Guidelines to the workload design in an RTVM
+############################################
+
+An RTOS developer must be aware of the differences between running applications on a native
+platform and a real time virtual machine (RTVM), especially issues impacting real time
+performance. For example, a real time thread should avoid using VM-Exit. If VM-Exit is used,
+the developer must account for an additional margin of CPU cycles for the added run time cost.
+
+This document provides some workload design guidelines when using an RTVM within the ACRN hypervisor.
+
+Run RTVM with dedicated resources/devices
+*****************************************
+
+For best practice, ACRN allocates dedicated CPU, memory resources, and cache resources (with
+Cache Allocations Technology, i.e. CAT) to RTVMs. For best real time performance of I/O devices,
+we recommend using dedicated (pass-thru) PCIe devices to avoid VM-Exit at run time.
+
+NOTE: The configuration space for pass-thru PCI devices is still emulated and the access will
+trigger a VM-Exit.
```

Comment:
```suggestion
trigger a ``vmexit``.
```

Reply:
SDM uses "VM-Exit", so our doc is better to stay with